examples/tensorflow/train/crnn/README.md

```diff
@@ -0,0 +1,106 @@
+# CRNN_Tensorflow example 
+Use tensorflow to implement a Deep Neural Network for scene text recognition mainly based on the paper "An End-to-End Trainable Neural Network for Image-based Sequence Recognition and Its Application to Scene Text Recognition".You can refer to their paper for details http://arxiv.org/abs/1507.05717. Thanks for the author Baoguang Shi.
+This model consists of a CNN stage, RNN stage and CTC loss for scene text recognition task.
+
+## Original Example
+Code here is an example of how to run CRNN_Tensorflow example on UAI Train platform.The original code can see here:https://github.com/MaybeShewill-CV/CRNN_Tensorflow/.<br>
+
+## Preparing  Data 
+### Download dataset
+The model is trained on a subet of [Synth 90k](http://www.robots.ox.ac.uk/~vgg/data/text/). You should download and unzip the dataset to get trainning data.
+### Preparing tfrecords
+Firstly,you need to supply a txt file named sample.txt to specify the relative path to the image data dir and it's corresponding text label. For example
```

Comment:
First, you should provide a txt file named sample.txt to specify the relative path to the image data dir and its corresponding text label.